encoding/gproperties/gproperties_z_unit_test.go

```diff
@@ -0,0 +1,53 @@
+package gproperties_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/gogf/gf/v2/encoding/gproperties"
+	"github.com/gogf/gf/v2/frame/g"
+	"github.com/gogf/gf/v2/test/gtest"
+)
+
+var pStr string = `
+# template
+data = "/home/www/templates/"
+# MySQL 
+sql.disk.0  = 127.0.0.1:6379,0
+sql.cache.0 = 127.0.0.1:6379,1=
+sql.cache.1=0
+sql.disk.a = 10
+`
+
+func TestDecode(t *testing.T) {
```

Comment:
Please add more UT cases to guarantee `90%` code coverage for new package.
Please refer: https://github.com/magiconair/properties/blob/main/properties_test.go

Reply:
all interfaces has ben add UT

Reply:
The code coverage is just `65.12%`, please refer to: https://codecov.io/gh/gogf/gf/pull/1806?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=gogf
Is there's difficulty in code coverage, please guarantee at least `80%` code coverage.
![image](https://user-images.githubusercontent.com/26347176/167781728-fafe364b-25bd-4a0f-99b8-f4636c0b4608.png)

Reply:
UT  added.